internal/tiltfile/tiltfile_state.go

```diff
@@ -82,25 +89,36 @@ const (
 	kustomizeN    = "kustomize"
 )
 
-func (s *tiltfileState) builtins() starlark.StringDict {
+func addBuiltin(r starlark.StringDict, name string, fn func(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error)) starlark.StringDict {
+	r[name] = starlark.NewBuiltin(name, fn)
+
+	return r
+}
+
+func (s *tiltfileState) builtins(thread *starlark.Thread) starlark.StringDict {
 	r := make(starlark.StringDict)
-	add := func(name string, fn func(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error)) {
-		r[name] = starlark.NewBuiltin(name, fn)
-	}
 
-	add(dockerComposeN, s.dockerCompose)
+	r = addBuiltin(r, localN, s.local)
+	r = addBuiltin(r, readFileN, s.skylarkReadFile)
+
+	// TODO(dmiller) this really only needs to be done once, on startup
+	r, err := starlark.ExecFile(thread, "helm.builtin", helmFunc, r)
```

Comment:
can we use a dummy thread here? i don't see a good reason to pass in an external thread, do you?

Reply:
also probably don't need to re-exec this every time we exec a new tiltfile